testingfacade/APIs/schemas/QuestionSchema.json

```diff
@@ -0,0 +1,167 @@
+                            {
+                                "answer_id": "answer_2",
+                                "label": "s3/wright",
+                                "description": "Mock sender 3",
+                                "id": "71ce775c-cdc5-479a-bd29-87c081905ecf",
+                                "answer_str": "s3/wright (Mock sender 3, 71ce775c-cdc5-479a-bd29-87c081905ecf)"
+                            }
+                        ]
+                    ],
+                    "items": {
+                        "description": "Individual answer details, the label, description and id of the relevant resource and an associated answer_id",
+                        "type": "object",
+                        "required": [
+                            "answer_id",
+                            "label",
+                            "description",
+                            "id",
+                            "answer_str"
+                        ],
+                        "properties": {
+                            "answer_id": {
+                                "type": "string",
+                                "description": "ID unique to each answer within a question. This is the value to be returned for validation."
+                            },
+                            "label": {
+                                "type": "string",
+                                "description": "Resource label as might be displayed on the NCuT"
+                            },
+                            "description": {
+                                "type": "string",
+                                "description": "Resource description as might be displayed on the NCuT"
+                            },
+                            "id": {
+                                "type": "string",
+                                "description": "Resource ID as might be displayed on the NCuT"
+                            },
+                            "answer_str": {
+                                "type": "string",
+                                "description": "String combining all the resource details for display on the testing facade"
+                            }
+                        }
+                    }
+                }
+            ]
+        },
+        "time_sent": {
+            "type": "number",
+            "description": "Time the question was sent, unix timestamp in seconds"
+        },
+        "timeout": {
+            "type": "number",
+            "description": "Time in seconds the test suite will wait to receive an answer to a particular question",
+            "default": 600
+        },
+        "answer_uri": {
+            "type": "string",
+            "description": "URL of the test suite API endpoint to send the POST request with answers"
+        },
+        "metadata": {
+            "anyOf": [
+                {
+                    "type": "null"
+                },
+                {
```

Comment:
This part of the `metadata` schema isn't right.

Reply:
This still needs `"type": "object"` and for the properties to be inside a `properties` property, doesn't it?

Do you want to allow only a single sender _or_ a single receiver in the metadata? Not one (or several) of each?